utils/outputwriter/outputcontent_test.go

```diff
@@ -190,52 +190,52 @@ func TestGetPRSummaryContent(t *testing.T) {
 			cases: []OutputTestCase{
 				{
 					name:               "Pull Request not entitled (Standard output)",
-					writer:             &StandardOutput{},
+					writer:             GetTestWriter(WriterPR, false, false, false, ""),
```

Comment:
Instead of repeated "true", "false", "true"...
Let's provide MarkdownOutput as the parameter to GetTestWriter.